### PR TITLE
Fixing scheduling adding logging

### PR DESCRIPTION
**Pull request by lakinwecker**
Even more tracing.

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -153,7 +153,7 @@ rules:
   max-len: 0
   max-nested-callbacks: 0
   max-params: 0
-  max-statements: [2, 50]
+  max-statements: [2, 60]
```
Comment (endrawes0) [inline .eslintrc]: Just 60? I expected to see an order of magnitude ;-)

`max-statements: [2, 500]`

File: src/chesster.js [hunk 2]
```diff
@@ -529,34 +529,47 @@ chesster.on({
     middleware: [slack.withLeague]
 },
 function(bot, message) {
+    var channel = channels.byId[message.channel];
```
Comment (endrawes0) [inline src/chesster.js]: Sometimes you have to make a mess worse before it gets better...

**Comment by endrawes0** [comment]
Looks good

File: .eslintrc (hunk 1 above)
Comment (lakinwecker) [inline .eslintrc]: lol.

**Pull request merged**